src/LibraryManager.Vsix/UI/InstallDialog.xaml.cs

```diff
@@ -89,15 +90,39 @@ public Task<CompletionSet> TargetLocationSearch(string searchText, int caretPosi
 
             foreach (Tuple<string, string> completion in completions)
             {
-                CompletionItem completionItem = new CompletionItem
+                bool addItem = false;
+                string insertionText = completion.Item2;
+
+                if (searchText.Equals(insertionText, StringComparison.OrdinalIgnoreCase))
+                {
+                    addItem = true;
+                }
+                else if (insertionText.StartsWith(searchText, StringComparison.OrdinalIgnoreCase))
+                {
+                   addItem = true;
+                }
+                else if (insertionText.IndexOf(searchText, StringComparison.OrdinalIgnoreCase) > -1)
                 {
-                    DisplayText = completion.Item1,
-                    InsertionText = completion.Item2,
-                };
+                    addItem = true;
+                }
+
+                if (addItem)
+                {
+                    CompletionItem completionItem = new CompletionItem
+                    {
+                        DisplayText = completion.Item1,
+                        InsertionText = completion.Item2,
+                    };
 
-                completionItems.Add(completionItem);
+                    completionItems.Add(completionItem);
+                }
             }
 
+           completionItems = completionItems.Where(t => t.InsertionText.ToLower()
+                                            .Contains(searchText.ToLower()))
```

Comment:
Actually, I'm not clear on what this Where clause gives you after the filtering applied in the foreach